### PR TITLE
CORE-12165: key rotation no limit

### PR DESCRIPTION
**Pull request by dickon**
* bump API reference to 5.2
* reduce trace level
* change REST path to 5.2
* drop simulate, timeToLive and limit parameters (due to YAGNI)

### CODE REVIEW

File: components/crypto/crypto-softhsm-impl/src/main/kotlin/net/corda/crypto/softhsm/WrappingRepository.kt [hunk 1]
```diff
@@ -41,7 +39,7 @@ interface WrappingRepository : Closeable {
     fun findKey(alias: String): WrappingKeyInfo?
 
     /**
-     * Find a wrapping key in the database
+     * Find a wrapping key in the database`
```
Comment (anien) [inline components/crypto/crypto-softhsm-impl/src/main/kotlin/net/corda/crypto/softhsm/WrappingRepository.kt]: Revert the change

Comment (dickon) [reply components/crypto/crypto-softhsm-impl/src/main/kotlin/net/corda/crypto/softhsm/WrappingRepository.kt]: done

**Review by anien — DISMISSED** [review]
LGTM.

**Review by anien — APPROVED** [review]
LGTM.
